authentik/core/sources/flow_manager.py

```diff
@@ -332,7 +350,34 @@ def handle_enroll(
                 )
             ],
             **{
-                PLAN_CONTEXT_PROMPT: delete_none_values(self.enroll_info),
+                PLAN_CONTEXT_PROMPT: delete_none_values(self.enroll_properties),
                 PLAN_CONTEXT_USER_PATH: self.source.get_user_path(),
             },
         )
+
+
+class UserUpdateStage(StageView):
```

Comment:
rely on user_write stage on enrollment flow (which is already a requirement) and dynamically insert a stage on login flow (should eventually be user_write but due to custom requirements can be a custom one for now)